### PR TITLE
Cockpit 301

### PR DESCRIPTION
**Pull request by jelly**
Preview: https://jelly.github.io/cockpit-project.github.io/blog/cockpit-301.html

### CODE REVIEW

File: _posts/2023-09-20-cockpit-301.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Networking: Add support for WireGuard
+
+The Networking page can now create and edit [WireGuard VPN](https://www.wireguard.com/) connections.
+
+![screenshot of add support for wireguard]({{ site.baseurl }}/images/301-networking-add-support-for-wireguard.png)
+
+![screenshot of add support for wireguard]({{ site.baseurl }}/images/301-networking-add-support-for-wireguard-2.png)
+
+_Many thanks to [Subho Ghosh](https://github.com/subhoghoshX) for adding this feature as part of his [Google Summer of Code](https://summerofcode.withgoogle.com/) project! and thanks to [Gil Obradors](https://github.com/gil-obradors) for his initial work._
```
Comment (martinpitt) [inline _posts/2023-09-20-cockpit-301.md:33]:
```suggestion
_Many thanks to [Subho Ghosh](https://github.com/subhoghoshX) for adding this feature as part of his [Google Summer of Code](https://summerofcode.withgoogle.com/) project! And thanks to [Gil Obradors](https://github.com/gil-obradors) for his initial work._
```

File: _posts/2023-09-20-cockpit-301.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Cockpit 301 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/301)
+* [Cockpit Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-15c074a031)
+* [Cockpit Fedora 39](https://bodhi.fedoraproject.org/updates/FEDORA-2023-22db412e5e)
+
+* [Cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/301)
```
Comment (martinpitt) [inline _posts/2023-09-20-cockpit-301.md:47]: :grin:

File: _posts/2023-09-20-cockpit-301.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+* [Cockpit-podman Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-751f8bb5d0)
+* [Cockpit-podman Fedora 39](https://bodhi.fedoraproject.org/updates/FEDORA-2023-5fb734ce86)
+
+* [Cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/301)
```
Comment (martinpitt) [inline _posts/2023-09-20-cockpit-301.md:51]: here as well

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks! Not a Garrett-level review of course, but found some errors.

**Review by allisonkarlitskaya — COMMENTED** [review]
The only things I found were already caught by Martin, so looks good to me, otherwise.

**Review by allisonkarlitskaya — APPROVED** [review]

**Comment by jelly** [comment]
> The only things I found were already caught by Martin, so looks good to me, otherwise.

Danke, just pushed fixed!

**Review by martinpitt — APPROVED** [review]

**Pull request merged**